resume/resume.org

```diff
@@ -53,16 +60,25 @@ Achievements
 Power Home Remodeling Group sells a large range of home remodeling products to home owners across the America from 16 branch offices.
 
 Responsibilities
-- Build, manage, maintain, and automate Power's cloud infrastructure consisting of Kubernetes, Vmware, NetApp and F5 using Ansible, Puppet, Docker and Terraform.
+- Build, manage, maintain, and automate Power's on premise infrastructure
```

Comment:
Minor style points. 

There's an odd tense change in the overall picture of the resume. For your current employer, you use the past-tense, but for this prior employer you use the present-tense. It's subtle, but mentally-foggy for the reader. Had you considered using a consistent tense throughout? I'm guilty of this tense change in my resume as well. I usually fall into the English trap of using present-tense for my current employer, then past-tense for all previous work experience. However, I've been toying with the idea of using present-tense throughout my resume. My current opinion is that this tends to create a tone that's a bit easier to digest. But again, just opinions and considerations. 

```suggestion
- Build and support Power's on premise infrastructure 
```

Reply:
I hadn't noticed but this is exactly what I do as well.
Present tense does makes for easier reading and saves editing every time you change jobs. :)